hikari/events/typing_events.py

```diff
@@ -171,7 +171,7 @@ class GuildTypingEvent(TypingEvent):
     """
 
     @property
-    def channel(self) -> typing.Union[channels.GuildTextChannel, channels.GuildNewsChannel]:
+    def channel(self) -> typing.Union[channels.GuildTextChannel, channels.GuildNewsChannel, None]:
```

Comment:
These are getting fixed in #412

Reply:
Don't see that file in its diff

Reply:
Nvm, it was implemented in #404 (which is missing the assert to allow `None`, got to fix that)